### DISCUSSION TITLE
Nice tiger look :-P

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#5147] [General]
Hi developers,

guess it is in process :-)

![grafik](https://user-images.githubusercontent.com/76531893/122525644-bd159480-d019-11eb-81c2-62d6f7b7b87d.png)

Greets Klaus

**Comment by tracernz** [comment]
Are you using a livery here? Haven't seen that before.

**Reply by CptBayer** [reply]
Yes, might be the issue... Will change to the one before and will report ... 

greets Klaus

**Reply by CptBayer** [reply]
ok, seems to be the American Airlines Piedmont livery ...... switched to another,no tiger look ..

greets Klaus

p.s. your free text option is so cool !!
